cloud-controlplane/cloud-controlplane.yaml

```
@@ -5684,7 +5690,8 @@ tags:
     name: Resource Groups
   - description: Manage [Redpanda Serverless](https://docs.redpanda.com/redpanda-cloud/get-started/cloud-overview/#redpanda-cloud-cluster-types) clusters. For detailed steps, see [Use the Control Plane API with Serverless](https://docs.redpanda.com/redpanda-cloud/manage/api/cloud-serverless-controlplane-api/).
     name: Serverless Clusters
-  - name: ServerlessPrivateLinkService
+  - description: Manage Redpanda Serverless Private Links. For detailed steps and information, see [Use the Control Plane API](https://docs.redpanda.com/redpanda-cloud/manage/api/controlplane/).
```

```suggestion
  - description: Manage Redpanda Serverless Private Links.
```
Remove link because there isn't any actual guidance yet for private links in Serverless. Timeline for docs is tbd (Q3/Q4).